### PR TITLE
Initial smokehouse integration

### PR DESCRIPTION
**Pull request by jburger424**
This is a bit hacky right now, but should only be a temporary workaround until some small modifications are made to Lighthouse to remove hard-coded paths.

Does not have full coverage right now, but should be a good starting point to build on.

### CODE REVIEW

File: lighthouse-plugin-publisher-ads/test/smoke/run-smoke.js [hunk 1]
```diff
@@ -0,0 +1,219 @@
+// Copyright 2019 Google LLC
```
Comment (jburger424) [inline lighthouse-plugin-publisher-ads/test/smoke/run-smoke.js]: File forked from https://github.com/GoogleChrome/lighthouse/blob/a83133f04321519edc60de4790a1aa8ee5b9afe5/lighthouse-cli/test/smokehouse/run-smoke.js

File: lighthouse-plugin-publisher-ads/test/smoke/fixtures/lazy-load.html [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    </div>
+
+    <div id="div9" style="width: 728; height: 90">
+      <script>
+        googletag.cmd.push(function() { googletag.display('div9'); });
+      </script>
+    </div>
+
+  </body>
+</html>
```
Comment (jonkeller) [inline lighthouse-plugin-publisher-ads/test/smoke/fixtures/lazy-load.html]: Nit: add newline

File: lighthouse-plugin-publisher-ads/test/smoke/fixtures/script-injected.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+      <script>
+          googletag.cmd.push(function() {
+            googletag.defineSlot('/6355419/Travel', [728, 90], `div0`)
+                  .setTargeting("test","infinitescroll")
+                  .addService(googletag.pubads());
+            googletag.pubads().enableSingleRequest();
+            googletag.enableServices();
+          });
+      </script>
+    <h2>Blocked Ad</h2>
```
Comment (jonkeller) [inline lighthouse-plugin-publisher-ads/test/smoke/fixtures/script-injected.html]: Is this h2 intentional?

File: lighthouse-plugin-publisher-ads/test/smoke/fixtures/long-tasks.html [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+              .addService(googletag.pubads());
+        googletag.pubads().enableSingleRequest();
+        googletag.enableServices();
+      });
+      }
+    </script>
+    <script>googletag = window.googletag || {cmd: []};</script>
+    <script async src="https://www.googletagservices.com/tag/js/gpt.js"></script>
+  <body>
+    <h2>Blocked Ad</h2>
```
Comment (jonkeller) [inline lighthouse-plugin-publisher-ads/test/smoke/fixtures/long-tasks.html]: Is this h2 intentional?

File: lighthouse-plugin-publisher-ads/test/smoke/fixtures/top-of-viewport.html [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+-->
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Lazy Load</title>
```
Comment (jonkeller) [inline lighthouse-plugin-publisher-ads/test/smoke/fixtures/top-of-viewport.html]: Nit: update title

File: lighthouse-plugin-publisher-ads/test/smoke/fixtures/top-of-viewport.html [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+        Suspendisse sed auctor mauris, sit amet faucibus neque. Praesent commodo 
+        tortor et lorem imperdiet, ac interdum massa molestie. Aliquam in 
+        bibendum sem. Duis bibendum ex dolor, et molestie sem aliquam non. Duis 
+        eu risus nec lectus maximus imperdiet. In mi velit, varius sit amet enim 
+        a, semper tristique sapien. Nulla in leo sed nibh dictum pharetra rhoncus 
+        id lacus.
+      </p>
+
+  </body>
+</html>
```
Comment (jonkeller) [inline lighthouse-plugin-publisher-ads/test/smoke/fixtures/top-of-viewport.html]: Nit: newline

File: lighthouse-plugin-publisher-ads/test/smoke/run-smoke.js [hunk 7]
```diff
@@ -0,0 +1,219 @@
...
+    `mv ${CORE_DFNS_PATH}.backup ${CORE_DFNS_PATH}`,
+    // Remove local and core `.tmp` dirs.
+    `rm -r .tmp`,
+    `rm ${TMP_DIR}`,
+    // Remove symlinked `lighthouse-cli`.
+    `rm lighthouse-cli`,
+  ];
+  execSync(cmd.join(' && '));
+}
+
```
Comment (jonkeller) [inline lighthouse-plugin-publisher-ads/test/smoke/run-smoke.js]: Given that this file is forked...lemme know if I need to read anything below here?

**Pull request merged**